core/src/test/java/com/adobe/aio/auth/OAuthContextTest.java

```diff
@@ -8,7 +8,7 @@
 
 public class OAuthContextTest {
 
-  private static final String TEST_PROPERTIES = "workspace.properties";
+  private static final String TEST_PROPERTIES = "workspace.oauth.properties";
```

Comment:
you could also revert all your `resources` changes... right ?
they are not compulsory...

Reply:
Review Comments Incorporated